src/test/java/teammates/test/cases/browsertests/FeedbackRubricQuestionUiTest.java

```diff
@@ -68,36 +68,35 @@ private void testInstructorResultsPage() throws Exception {
         InstructorFeedbackResultsPage instructorResultsPage =
                 loginToInstructorFeedbackResultsPageWithViewType("teammates.test.instructor", "openSession2",
                                                                  false, "question");
-        instructorResultsPage.waitForPanelsToExpand();
-
+        clickAjaxLoadedPanelAndWaitForExpansion(instructorResultsPage, "panelHeading-1", "ajax_auto");
```

Comment:
Add a comment before ```clickAjaxLoadedPanelAndWaitForExpansion``` documenting that this is the test that tests the AJAX functionality so that it won't be accidentally removed